lib/route_translator/extensions/mapper.rb

```diff
@@ -41,6 +41,18 @@ def add_route(action, controller, options, original_path, to, via, formatted, an
           @set.add_route(mapping, ast, as, anchor)
         end
       end
+
+      private
+
+      def define_generate_prefix(app, name)
+        if @localized
+          RouteTranslator::Translator.available_locales.each do |locale|
+            super(app, "#{name}_#{locale.to_s.underscore}")
+          end
+        else
+          super
```

Comment:
Added a test for it https://github.com/enriclluelles/route_translator/pull/154/commits/163f0bb576c98d3a89feadf3f0e540ecaf185f0e.